sourcecode/hub/resources/views/user/my-account.blade.php

```diff
@@ -0,0 +1,25 @@
+<x-layout>
+    <x-slot name="title">{{ trans('messages.my-account') }}</x-slot>
+
+    <h6>{{ trans('messages.change-profile-name') }}</h6>
+    @if(session('success'))
```

Comment:
The base template can show an alert, so having code for showing one here isn't necessary.